dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -107,6 +112,15 @@ object GlobalRum {
 
     // region Internal
 
+    internal fun addUserInteraction() {
```

Comment:
Is this intended to be used by all RUM event sources to keep the active session alive? I don't see any usage of this in this PR beside tests 🤔.

Reply:
It's meant to be used only through UserActions only. But indeed, forgot to do that in the RumMonitor